final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +75,68 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
-
+scary_monster = {
+  'name' => 'Deforestation',
+  'smell' => 'cedar',
+  'weight' => 3000,
+  'citiesDestroyed' => ['Seattle', 'Denver', 'Portland'],
+  'luckyNumbers' => [13, 77, 99],
+  'address' => {
+    "number" => 1810,
+    "street" => "Julian",
+    "state" => "Colorado",
+    "zip" => 80204
+    }
+  }
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
-
+#   - energy_level = 50
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  attr_reader :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age)
+    self.name = name
+    self.super_power = super_power
+    self.age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+  end
+
+  def say_name
+    p name
+  end
+
+  def maximize_energy
+    @energy_level = 1000
+  end
+
+  def gain_power(level)
+    @power_level = power_level + level
+  end
+end
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
 
 # - Create 2 instances of your SuperHero class
-
+sharkboy = SuperHero.new("Sharkboy", "sharp teeth", "19")
+cloakman = SuperHero.new("Cloakman", "invisible", "52")
 
 # Reflection
 # What parts were most difficult about this exerise?
+# Functions were the most difficult part for me
 
 # What parts felt most comfortable to you?
+#the easy stuff like making/manipulating variables
 
 # What skills do you need to continue to practice before starting Mod 1?
-
+#I need to practice functions! 
```

Comment:
A great thing to practice! You have a nice dynamic function above, that takes an argument. I'd experiment with more simple functions like that! Ones that take an argument, and use that in some way for the output.